src/point_cloud_editor/render_widget.cc

```diff
@@ -36,6 +36,7 @@
 
 #include "camera/camera_pinhole.h"
 #include "opengl/opengl_util.h"
+#include <Eigen/StdVector>
```

Comment:
Minor: The include order in the codebase should (hopefully) be like this, with a free line between each block:

```c++
// for source files, #include "" of the corresponding header

// system #includes <>

// external library #includes <>

// #includes "" within the same project
```

So, for consistency, the Eigen include should go to the block above.